keyboards/handwired/ziyoulang_k3_mod/keymaps/via/rules.mk

```
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+# Blue Pill STM32F103C6T6 which only has 22KB of available space.
+# In order to reduce the size of Firmware, disabling mousekey and via LTO were necessary.
+MOUSEKEY_ENABLE = no
+LTO_ENABLE = yes
```

It might be worth enabling lto at the keyboard level, simply because of the low firmware capacity. Otherwise: 

```suggestion
MOUSEKEY_ENABLE = no
LTO_ENABLE = yes
```